sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
 # Release History
 
-## 1.2.4 (Unreleased)
+## 1.2.5 (Unreleased)
 
-## 1.2.4-beta.2 (Unreleased)
+## 1.2.5-beta.1 (Unreleased) (Continues from 1.2.4-beta.1)
```

Comment:
I would recommend adding this note on 1.2.4-beta.1 in the text under the header rather than the header itself. Same for 1.2.4
Also... since we are introducing a breaking change... shouldnt the major version be updated?

Reply:
> major

Scary! 😅

That is true though. I'll do the change in this PR.